katas/content/superposition/three_states_two_qubits_phases/index.md

```diff
@@ -0,0 +1,5 @@
+**Inputs:**
+
+1. Two qubits in the $|00\rangle$ state (stored in an array of length 2).
+
+**Goal:** Change the state to $\frac{1}{\sqrt{3}} \big( |00\rangle +  |01\rangle + |10\rangle \big)$ where $\omega$ is $e^{2\pi\\i/3}$.
```

Comment:
Omega missing in state description, and \\i command is not recognized by the parser. Do you build the playground locally? Seeing the new task built should help find such issues.
```suggestion
**Goal:** Change the state to $\frac{1}{\sqrt{3}} \big( |00\rangle + \omega |01\rangle + \omega^2 |10\rangle \big)$, where $\omega = e^{2\pi i/3}$.
```

Reply:
Ah, yeah I've been building it using `python ./build.py  --wasm --npm --play`. I'm using VS Code so had been checking the markdown using the Preview tool, I see now when viewing it via the playground it is not correct! Will check there in future